kalite/distributed/templates/distributed/partials/khan_dummy_channel_translation.html

```diff
@@ -0,0 +1,7 @@
+<!-- TODO BLOCKER: 0.14 (rtibbles) this is dummy text to pull in Khan Academy channel data for translation. This is not ideal !-->
+
+{% trans "KA Lite" %}
+{% trans "A free world-class education for anyone anywhere." %}
+{% trans "KA Lite is a lightweight web server for viewing and interacting with core Khan Academy content (videos and exercises) without needing an Internet connection." %}
+{% trans "CC-BY-NC-SA" %}
```

Comment:
Do we really want this translated? I'd imagine this set of strings to indicate the type of CC license is standard everywhere, and it would just be confusing if it's translated.